rohit/lec-05 JNI/classes/Permutation.c

```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
+#include "com_rohit_jni_Permutation.h"
+
+void permute(int *arr, int idx, int len) {
+	if(idx >= len - 1) {
+		printf("[");
+		for(int i = 0; i < len - 1; i++)
+			printf("%d, ", arr[i]);
+		if(len > 0)
+			printf("%d", arr[len - 1]);
+		printf("]\n");
+		return;
+	}
```

Comment:
This work is really nice. It would have been much better if you would have passed the results back to java before printing.